turbo/shards/state_cache.go

```diff
@@ -0,0 +1,635 @@
+package shards
+
+import (
+	"bytes"
+	"container/heap"
+	"fmt"
+
+	"github.com/google/btree"
+	"github.com/holiman/uint256"
+	"github.com/ledgerwatch/turbo-geth/common"
+	"github.com/ledgerwatch/turbo-geth/core/types/accounts"
+)
+
+// LRU state cache consists of two structures - B-Tree and binary heap
+// Every element is marked either as Read, Updated, or Deleted via flags
+
+const (
+	ModifiedFlag uint16 = 1
+	DeletedFlag  uint16 = 2
+)
+
+//nolint:maligned
```

Comment:
To satisfy this linter: move fixed size fields to top of struct declaration. 
Move down fields of type `Account` and `[]byte` - their size is unknown.

Reply:
Thank you, fixed